libkbfs/interfaces.go

```diff
@@ -952,6 +974,12 @@ type BlockOps interface {
 	Archive(ctx context.Context, tlfID TlfID, ptrs []BlockPointer) error
 }
 
+// AuthTokenRefreshHandler defines a callback to be called when an auth token refresh
+// is needed.
+type AuthTokenRefreshHandler interface {
```

Comment:
Oh, yeah, forgot to fix the comment. This has to be here because of gomock's limitations (since we embed it everywhere).